### PR TITLE
Fix for #17: adds `full` attribute.

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -29,17 +29,26 @@
     <h1>&lt;iron-media-query&gt;</h1>
 
     <template is="dom-bind">
-      <iron-media-query query="(min-width: 600px)" query-matches="{{queryMatches}}"></iron-media-query>
+      <iron-media-query query="(min-width: 600px)" query-matches="{{wide}}"></iron-media-query>
+      <iron-media-query full query="print" query-matches="{{print}}"></iron-media-query>
 
-      <template is="dom-if" if="{{queryMatches}}">
+      <template is="dom-if" if="{{wide}}">
```
Comment (notwaldorf) [inline demo/index.html]: so happy about the `s/queryMatches/wide`. It was confusing people before :beers:

File: iron-media-query.html [hunk 2]
```diff
@@ -49,6 +49,15 @@
...
         observer: 'queryChanged'
       },
+
+      /**
+       * If true, the query attribute is assumed to be a complete media query
+       * string rather than a single media feature.
+       */
+      full: {
+        type: Boolean,
+        value: false,
```
Comment (notwaldorf) [inline iron-media-query.html]: eeek, trailing comma

**Comment by notwaldorf** [comment]
@bicknellr This is awesome! One tiny nit and it's LGTM

File: iron-media-query.html (hunk 2 above)
Comment (bicknellr) [inline iron-media-query.html]: gone

**Pull request merged**